### PR TITLE
docs(authn/oauth): Add docs for --user-info-requirements.

### PR DESCRIPTION
**Pull request by stewchen**

### CODE REVIEW

File: setup/security/authentication/oauth/index.md
Comment (dorbin) [inline setup/security/authentication/oauth/index.md]: Maybe use an example that includes multiple key-value pairs, to make it even clearer to repeat the flag?

**Review by dorbin — COMMENTED** [review]
LGTM, after my one comment, and pending review by @ttomsu .

File: setup/security/authentication/oauth/index.md
Comment (ttomsu) [reply setup/security/authentication/oauth/index.md]: IIRC someone added the ability to use regex in those reqs also.

Comment (ttomsu) [reply setup/security/authentication/oauth/index.md]: Instead of `google.com` in the example, I'd say something like `my-org.net` or something that indicates the user's domain.

Comment (stewchen) [reply setup/security/authentication/oauth/index.md]: Done, added multiple values, added notes on regex.

**Review by ttomsu — APPROVED** [review]

**Pull request merged**
